pints/_mcmc/__init__.py

```diff
@@ -399,6 +446,8 @@ def chains(self):
         """
         # Note: Not copying this, for efficiency. At this point we're done with
         # the chains, so nothing will go wrong if the user messes the array up.
+        # Note: Any inverse transform for parameters should be performed before
+        # stored to memory.
```

Comment:
- stored --> storing
- add "or logging to screen"

Reply:
Done